website/content/docs/integrations/consul-integration.mdx

```diff
@@ -167,55 +189,6 @@ service_prefix "" {
 }
 ```
 
-Using the same configuration file shown above, create the Consul Auth Method to
```

Comment:
I'd probably add a reference back to the auth method and binding rule we created earlier just to make it extra clear how this policy is getting associated with jobs. Something like:

> The binding rule you created above for Nomad namespaces will map workloads to the role with this policy.

Alternately, you could move the binding rule creation here so that it's more directly associated with the role.